src/toil/batchSystems/options.py

```diff
@@ -17,6 +17,10 @@
 from argparse import ArgumentParser, _ArgumentGroup
 from typing import Any, Callable, List, Optional, TypeVar, Union, cast
 
+from configargparse import ArgParser
+
+SYS_MAX_SIZE = 9223372036854775807
```

Comment:
I think the original defaults were sys.maxsize, but similar defaults in common.py use SYS_MAX_SIZE. This option was duplicated from common.py since python complained about circular imports. I can probably import SYS_MAX_SIZE within the add_all_batchsystem_options function, or switch to using sys.maxsize, but I would like the default values to be static.